src/dataStore/DripsDataStore.sol

```diff
@@ -0,0 +1,240 @@
+        bytes memory data = abi.encode(splits);
+        hash = keccak256(data);
+        if (_splitsPointers[hash] == address(0)) {
+            _splitsPointers[hash] = _store(data);
+            emit SplitsStored(hash);
+        }
+    }
+
+    /// @notice Checks if the list of splits receivers is stored.
+    /// @param hash The hash of the list, see `storeSplits`.
+    /// @return isStored True if the list of splits receivers is stored.
+    /// If `hash` is `EMPTY_HASH`, which is `0`, always returns true.
+    function isSplitsStored(bytes32 hash) public view returns (bool isStored) {
+        if (hash == EMPTY_HASH) return true;
+        return _splitsPointers[hash] != address(0);
+    }
+
+    /// @notice Loads the list of splits receivers.
+    /// Reverts if the list with the given hash isn't stored, see `isSplitsStored`.
+    /// @param hash The hash of the list, see `storeSplits`.
+    /// @return splits The loaded list of splits receivers.
+    function loadSplits(bytes32 hash) external view returns (SplitsReceiver[] memory splits) {
+        if (hash == EMPTY_HASH) return splits;
+        _loadAndReturn(_splitsPointers[hash]);
+    }
+
+    /// @notice Store a list of account metadata.
+    /// Storing an already stored list is valid, and it doesn't do anything.
+    /// @param accountMetadata The account metadata list to store.
+    /// @return hash The hash of the stored list, that can be used to load it.
+    /// If `splits` is empty, it's `EMPTY_HASH`, which is `0`.
+    function storeAccountMetadata(AccountMetadata[] calldata accountMetadata)
+        external
+        returns (bytes32 hash)
+    {
+        if (accountMetadata.length == 0) return EMPTY_HASH;
+        bytes memory data = abi.encode(accountMetadata);
+        hash = keccak256(data);
+        if (_accountMetadataPointers[hash] == address(0)) {
+            _accountMetadataPointers[hash] = _store(data);
+            emit AccountMetadataStored(hash);
+        }
+    }
+
+    /// @notice Checks if the list of account metadata is stored.
+    /// @param hash The hash of the list, see `storeSplits`.
+    /// @return isStored True if the list of account metadata is stored.
+    /// If `hash` is `EMPTY_HASH`, which is `0`, always returns true.
+    function isAccountMetadataStored(bytes32 hash) public view returns (bool isStored) {
+        if (hash == EMPTY_HASH) return true;
+        return _accountMetadataPointers[hash] != address(0);
+    }
+
+    /// @notice Loads the list of account metadata.
+    /// Reverts if the list with the given hash isn't stored, see `isSplitsStored`.
+    /// @param hash The hash of the list, see `storeSplits`.
+    /// @return accountMetadata The loaded list of account metadata.
+    function loadAccountMetadata(bytes32 hash)
+        external
+        view
+        returns (AccountMetadata[] memory accountMetadata)
+    {
+        if (hash == EMPTY_HASH) return accountMetadata;
+        _loadAndReturn(_accountMetadataPointers[hash]);
```

Comment:
okay we don't need a `return _loadAndReturn(...);` here because 
```
/// and the loaded data is returned as the external function's return data.
````